index.js

```diff
@@ -41,18 +46,29 @@
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 4", "color: red") 
 
-// console.log(sameSameButDifferent(5, 5)) 
-// // => "same same"
-//
-// console.log(sameSameButDifferent(123, "123")) 
-// // => "same same (but different)"
-//
-// console.log(sameSameButDifferent(5, 7)) 
-// // => "different"
-//
-// console.log(sameSameButDifferent(123, "122")) 
-// // => "different"
-// console.log("%c----------", "color: red") 
+function sameSameButDifferent(num, maybeNum) {
+  num === maybeNum
+  console.log("same same")
+  num == maybeNum
+  console.log("same same (but differenct)")
+  num != maybeNum
+  console.log("different")
+  num !== maybeNum
+  console.log("different")
+}
```

Comment:
You've got the right idea in terms of using `==` and `===`, just make sure to use that along with some control flow (if/else), and the `return` keyword instead of `console.log` like this to get the right value returned:

```js
function sameSameButDifferent(num, maybeNum) {
  if (num === maybeNum) {
    return "same same"
  } else if (num === parseInt(maybeNum)) {
    return "same same (but different)"
  } else {
    return "different"
  }
}
```